help_to_heat/frontdoor/views.py

```diff
@@ -336,6 +336,22 @@ def handle_post(self, request, session_id, page_name, data, is_change_page):
 
         return redirect("frontdoor:page", session_id=session_id, page_name=next_page_name)
 
+    def get_prev_next_urls(self, session_id, page_name):
+        session_data = interface.api.session.get_session(session_id)
+        request_supplier = session_data.get("supplier")
+        prev_page_url, next_page_url = super().get_prev_next_urls(session_id, page_name)
+        if request_supplier == "Bulb, now part of Octopus Energy":
```

Comment:
I think I'd go one further than Sammy here, and have a dictionary of `acquired_supplier_warning_pages` say:
```
acquired_supplier_warning_pages = {
    "Bulb, now part of Octopus Energy": "bulb-warning-page",
    ...
}
```

Then we'd only need one
```
if request_supplier in acquired_supplier_warning_pages:
    prev_page_url = reverse(
                "frontdoor:page", kwargs=dict(session_id=session_id, page_name=acquired_supplier_warning_pages[request_supplier])
    )
```
rather than a long chain if `elif`s (that may get longer, who knows).